controls/repository.sp

```diff
@@ -51,10 +52,8 @@ benchmark "repository_checks" {
 benchmark "repository_mod_checks" {
   title = "GitHub Mod Repository Checks"
   children = [
-    # TODO: Re-add control once v3 hydrate data works
-    #control.repository_mod_has_mandatory_topics,
-    # TODO: Re-add control after testing API limits
-    #control.repository_mod_uses_monotonic_versioning,
+    control.repository_mod_has_mandatory_topics,
```

Comment:
Did these controls throttle at all, or seem to slow down the overall execution (since they use v3 API hydrate data)?